kyuubi-common/src/main/scala/org/apache/kyuubi/util/KyuubiHadoopUtils.scala

```diff
@@ -60,11 +60,11 @@ object KyuubiHadoopUtils extends Logging {
     val byteStream = new ByteArrayOutputStream
     creds.writeTokenStorageToStream(new DataOutputStream(byteStream))
 
-    Base64.getMimeEncoder.encodeToString(byteStream.toByteArray)
```

Comment:
newline cannot be parsed by flink command, so I changed it to `Base64.getEncoder`

Reply:
please change it in an independent PR. and I searched the codebase, seems there are other places call `Base64.getMimeEncoder`, should we change them together?

Reply:
> please change it in an independent PR. and I searched the codebase, seems there are other places call `Base64.getMimeEncoder`, should we change them together?

reverted and send a new pr #6408